test_scripts/API/Restructuring_OnResetTimeout/011_BC_OnResetTimeout_for_one_interface_RPS_of_several.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC_1 for several interfaces is requested by mobile app
+-- 2) SDL sends Interface_1.RPC_1 and Interface_2.RPC_1
+-- 3) Some time after receiving requests  on HMI is passed
+-- 4) HMI sends BC.OnResetTimeout(resetPeriod = 6000) to SDL for request on Interface_1
+-- 5) HMI does not respond to both request
+-- SDL does:
+-- Respond in 10 seconds with GENERIC_ERROR resultCode to mobile app.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function PerformInteraction()
```

Comment:
Please use here another RPC, for example AddCommand, because there is so problems with timeouts for PI in case there are no response to both requests with type BOTH. interactionMode = "VR_ONLY" does not satisfy the conditions, SDL does not handle IU.PI in this case